content/en/docs/installation/armory-halyard.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Armory Halyard
+weight: 3
+---
+
+Armory Halyard is a versatile command line interface (CLI) to configure and deploy Spinnaker. It is made of a CLI that connects to a daemon.
+
+## Running in Docker
+
+Running Armory Halyard in Docker is convenient and portable. The daemon will need access to files and environment variables, such as:
+- Halyard's main configuration directory - make sure the daemon has write access to that directory
+- kubeconfig file for Spinnaker's installation cluster (usually `~/.kube/config`)
+- AWS profiles (usually `~/.aws`) if access to AWS is needed
+- Any other configuration files that reside on the Docker host
+
+> The Docker container expects to use `heptio-authenticator-aws` instead of `aws-iam-authenticator`
```

Comment:
Is this still true? I don't even think you can get the heptio authenticator post-acquisition

Reply:
I doubt it but I was just moving information around. A lot of stuff needs to be updated IMO.